docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/kustomize-set-image.md

```diff
@@ -17,8 +17,8 @@ to executing `kustomize edit set image`. This step is commonly followed by a
 | `path` | `string` | Y | Path to a directory containing a `kustomization.yaml` file. This path is relative to the temporary workspace that Kargo provisions for use by the promotion process.                                                                                                                                                                         |
 | `images` | `[]object` | N | The details of changes to be applied to the `kustomization.yaml` file. When left unspecified, all images from the Freight collection will be set in the Kustomization file. Unless there is an ambiguous image name (for example, due to two Warehouses subscribing to the same repository), which requires manual configuration.            |
 | `images[].image` | `string` | Y | Name/URL of the image being updated.                                                                                                                                                                                                                                                                                                         |
-| `images[].tag` | `string` | N | A tag naming a specific revision of `image`. Mutually exclusive with `digest`. If none of these are specified, the tag specified by a piece of Freight referencing `image` will be used as the value of this field.                                                                                                                          |
-| `images[].digest` | `string` | N | A digest naming a specific revision of `image`. Mutually exclusive with `tag`. If none of these are specified, the tag specified by a piece of Freight referencing `image` will be used as the value of `tag`.                                                                                                                               |
+| `images[].tag` | `string` | N | A tag naming a specific revision of `image`. Mutually exclusive with `digest`. Users must explicitly specify this field or use an [expression](../../60-reference-docs/40-expressions.md#imagefromrepourl-freightorigin).                                                                                                                          |
```

Comment:
The wording here is awkward (same of the other field, below). It doesn't sound right that "Users must explicitly specify this field _**OR**_ use an expression." Isn't it more the case that it would be explicitly specified _**often using**_ an expression?

"must explicitly specify" also makes this sound required, when the real requirement is that _one and only one_ of the `tag` or `digest` field is specified.